frontend/src/pages/listings/[...listingId].tsx

```diff
@@ -54,93 +47,48 @@ const ListingPage: NextPageWithLayout<InferGetServerSidePropsType<typeof getServ
-                sm={10}
-                direction="column"
-                alignItems="stretch"
-                spacing={4}
-                sx={(theme) => ({
-                  position: "relative",
-                  [theme.breakpoints.up("md")]: {
-                    marginTop: "-7%",
-                  },
-                })}
-              >
-                <Grid container item direction="row" alignItems="stretch" justifyContent="center" spacing={4}>
-                  <Hidden mdDown>
-                    <Grid item xs={4}>
-                      <InfoCard listing={data.listing} />
-                    </Grid>
-                  </Hidden>
-                  <Grid item xs>
-                    <TitleCard listing={data.listing} />
-                  </Grid>
-                </Grid>
-                <Grid item>
-                  <ListingBody>
-                    <ReactMarkdown components={markdownComponents}>
-                      {descriptionWithTitle(data.listing.description)}
-                    </ReactMarkdown>
-                  </ListingBody>
-                </Grid>
+          <Title
+            title={data.listing.title}
+            variant={data.listing.heroImageUrl ? "dark" : "normal"}
+            overline={data.listing.organization.name}
+            breadcrumbs={[
+              { href: "/", name: "Hjem" },
+              { href: "/listings", name: "Verv" },
+              { href: `/listings/${data.listing.id}`, name: data.listing.title },
+            ]}
+            bgImage={data.listing.heroImageUrl ?? undefined}
+            ImageProps={{
+              placeholder: "empty",
+              unoptimized: true,
+              layout: "fill",
+              objectPosition: "top",
+            }}
+          />
+          <Container sx={{ mb: 4 }}>
+            <Grid
+              container
+              direction="row-reverse"
+              justifyContent={{ xs: "center", sm: "space-between" }}
+              alignItems="flex-start"
+              spacing={4}
+            >
+              <Grid item xs={12} sm={6} md={5} direction="column">
+                <TitleCard listing={data.listing} />
+              </Grid>
+              <Grid item xs={12} sm={6} md={7}>
+                <ReactMarkdown components={markdownComponents}>
+                  {descriptionWithTitle(data.listing.description)}
+                </ReactMarkdown>
               </Grid>
             </Grid>
           </Container>
-          <Hidden mdUp>
-            <Paper
-              sx={{
-                position: "sticky",
-                bottom: 0,
-                padding: (theme) => theme.spacing(2),
-                zIndex: (theme) => theme.zIndex.snackbar,
-              }}
-            >
-              <Grid container direction="row" justifyContent="space-between" alignItems="center">
-                {data.listing.readMoreUrl && (
-                  <Grid item xs>
-                    <Link passHref href={data.listing.readMoreUrl}>
```

Comment:
Seems like we've dropped the `readMoreUrl` entirely?

Reply:
Hard to say where it should go now, now that the "Apply here" button is where it used to be... although I really like the new design. Could the two buttons perhaps be side by side? Only if the listing has its `readMoreUrl` field set, of course.

Reply:
Added it back, simply by adding a link to the organization's name, not the most visible, but it gets the job done.

Reply:
I like that!